prov/efa/configure.m4

```diff
@@ -113,6 +115,44 @@ AC_DEFUN([FI_EFA_CONFIGURE],[
 	AC_SUBST(efa_CPPFLAGS)
 	AC_SUBST(efa_LDFLAGS)
 	AC_SUBST(efa_LIBS)
+
+	AC_ARG_WITH([cmocka],
+		[AS_HELP_STRING([--with-cmocka=DIR],
+				[Provide path to where the Cmocka runtime libraries
+				 are installed.])],
+		[efa_unit_test=yes], [efa_unit_test=no])
+
+	if [test x"$efa_unit_test" = x"yes"] && [test x"$with_cmocka" != x"yes"]; then
```

Comment:
should be 

```
if [test x"$with_efa_unit_test" != x"no" && test -n "$with_efa_unit_test"].
```

e.g. user specified the `--with-efa-unit-test` and the value is not `no`.

Reply:
updated to be accurate